internal/dockerutil/fileretriever_test.go

```diff
@@ -0,0 +1,63 @@
+package dockerutil_test
+
+import (
+	"context"
+	"testing"
+
+	volumetypes "github.com/docker/docker/api/types/volume"
+	"github.com/strangelove-ventures/ibctest"
+	"github.com/strangelove-ventures/ibctest/internal/dockerutil"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zaptest"
+)
+
+func TestFileRetriever(t *testing.T) {
```

Comment:
These tests are fast enough for unit tests? i.e. doesn't need `testing.Short` check?